README.md

```diff
@@ -13,46 +13,47 @@ Znaczy [pull requestów](https://github.com/nurkiewicz/polski-w-it/pulls), nie t
 | default              | difoltowo           | domyślnie                             |
 | deploy               | zdiplojować         | wdrożyć                               |
 | developer            |                     | programista                           |
-| edge case            |                     | przypadek brzegowy                    |
-| event                |                     | zdarzenie                             |
-| feature              | ficzer              | funkcja                               |
-| feedback             |                     | informacja zwrotna                    |
+| edge case            | edż kejs            | przypadek brzegowy                    |
+| event                | iwent               | zdarzenie, wydarzenie                 |
+| feature              | funkcjonalność, ficzer| funkcja                             |
+| feedback             | fidbek              | informacja zwrotna                    |
 | fix                  | fiksnąć             | poprawić                              |
 | flow                 |                     | przepływ                              |
 | footer               |                     | stopka                                |
 | foreign key          | foren kij           | klucz obcy                            |
 | header               |                     | nagłówek                              |
 | impact               |                     | wpływ                                 |
-| issue                | isiu                | problem                               |
+| issue                | isiu                | kwestia, problem                      |
 | job                  |                     | zadanie                               |
 | label                | labelka             | etykieta                              |
 | latency              |                     | opóźnienie                            |
 | launch               | lancz               | uruchomić, wprowadzić na rynek        |
 | layout               | lejałt              | układ (np. elementów interfejsu)      |
 | lock                 | zalokować           | zablokować                            |
-| lunch                |                     | obiad, tyle że w Warszawie            |
+| launch               | lancz               | uruchomić, wprowadzić na rynek        |
+| lunch                | lancz               | obiad, tyle że w Warszawie            |
 | meeting              | miting              | spotkanie                             |
-| message              |                     | wiadomość                             |
+| message              | messedż             | wiadomość                             |
 | mockup               | mokap               | makieta                               |
-| paging               | paginacja           | stronicowanie                         |
+| paging               | paginacja, stronnicowanie| stronicowanie                    |
```

Comment:
Słuszna uwaga ze _stronnicowaniem_, 👍. Tylko tabelka się rozjechała i testy (https://github.com/nurkiewicz/polski-w-it/blob/d3a9b958cf812f346b04ae915e2fb7f65416ff5b/test_readme.py#L25-L30) nie przejdą.